File: src/main/java/redcrafter07/processed/block/tile_entities/capabilities/IFluidHandlerModifiable.java
package redcrafter07.processed.block.tile_entities.capabilities;

import net.neoforged.neoforge.fluids.FluidStack;
import net.neoforged.neoforge.fluids.capability.IFluidHandler;

public interface IFluidHandlerModifiable extends IFluidHandler {
    void setStackInTank(int tank, FluidStack stack);
}

File: src/main/kotlin/redcrafter07/processed/block/tile_entities/capabilities/IFluidHandlerModifiable.kt
package redcrafter07.processed.block.tile_entities.capabilities

import net.neoforged.neoforge.fluids.FluidStack
import net.neoforged.neoforge.fluids.capability.IFluidHandler

interface IFluidHandlerModifiable : IFluidHandler {
    fun setStackInTank(tank: Int, stack: FluidStack)
    fun getStackInTank(tank: Int): FluidStack
}